### PR TITLE
[9.0][FIX] account_invoice_merge: Wrong actions name

### PR DESCRIPTION
**Pull request by igallart**

### CODE REVIEW

File: account_invoice_merge/__openerp__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 {
     'name': 'Account Invoice Merge Wizard',
-    'version': '9.0.1.0.1',
+    'version': '9.0.1.0.2',
     'category': 'Finance',
     'author': "Elico Corp, Odoo Community Association (OCA)",
     'website': 'http://www.openerp.net.cn',
```
Comment (elicoidal) [inline account_invoice_merge/__openerp__.py]: please update to https://github.com/OCA/account-invoicing/ as we are at it :)

**Review by elicoidal — APPROVED** [review]
Make sense!
LGTM

File: account_invoice_merge/__openerp__.py (hunk 1 above)
Comment (igallart) [reply account_invoice_merge/__openerp__.py]: @elicoidal, what change do you mean? I do not understand you.

Comment (bodedra) [reply account_invoice_merge/__openerp__.py]: @igallart Change website key value from `http://www.openerp.net.cn` to `https://github.com/OCA/account-invoicing`

Comment (elicoidal) [reply account_invoice_merge/__openerp__.py]: Yes, this is what I meant :smile: !

**Review by bodedra — APPROVED** [review]

**Pull request merged**